mtags/src/main/scala/scala/meta/internal/mtags/OnDemandSymbolIndex.scala

```diff
@@ -145,6 +146,9 @@ final case class OnDemandSymbolIndex(
         val input = file.toInput
         val document = mtags.index(language, input)
         s.TextDocuments(List(document))
+      case "sc" =>
+        val res = semanticdb(file)
```

Comment:
Not 100% sure, but we use Scalameta parser here to create TextDocument on the fly. Shouldn't it also work with `sc` files, but a proper syntax? Or does Scalameta parser not support `.sc` files? Or are the changes so radical that we would need to have a custom one? 

I am worried there can be a race condition between when the file is indexed and senabticDB files is created.

Reply:
Just fixed that.